homeassistant/components/bmw_connected_drive/sensor.py

```diff
@@ -153,6 +154,15 @@ def convert_and_round(
         state_class=SensorStateClass.MEASUREMENT,
         is_available=lambda v: v.is_lsc_enabled and v.has_combustion_drivetrain,
     ),
+    BMWSensorEntityDescription(
+        key="activity",
+        translation_key="climate_status",
+        key_class="climate",
+        device_class=SensorDeviceClass.ENUM,
+        options=[s.lower() for s in ClimateActivityState if s != ClimateActivityState.UNKNOWN],
+        value=lambda x: x.lower() if x != ClimateActivityState.UNKNOWN else None,
```

Comment:
@rikroe is it ok or should we have here:

```suggestion
        value=lambda x: x.value.lower() if x != ClimateActivityState.UNKNOWN else None,
```

thanks

Reply:
This is perfect

Reply:
thanks, fix pushed, I got confused a bit there